frontend/src/metabase/static-viz/lib/waterfall.js

```diff
@@ -34,6 +34,31 @@ export const calculateWaterfallEntries = (data, accessors, showTotal) => {
   return entries;
 };
 
+export const calculateWaterfallSeriesForValues = (
```

Comment:
since `<Values />` expect `series` not `data` I need to construct `series` for waterfall chart, so I can use `<Values /> ` inside the waterfall chart component.

Otherwise, I need to duplicate `<Values />` that would just work for the waterfall chart. But I think it's harder to manage the component that way.

With minimal `<Values />` API change I can make it work with our waterfall chart. Hence this function.